wallet/src/actors/app/handlers/create_wallet.rs

```diff
@@ -9,7 +9,7 @@ use crate::types;
 #[derive(Debug, Serialize, Deserialize)]
 pub struct CreateWalletRequest {
     name: Option<String>,
-    caption: Option<String>,
+    description: Option<String>,
```

Comment:
As always, we must remember to change this in sheikah.

Reply:
This is being worked by @Tommytrg  in https://github.com/witnet/sheikah/issues/1486 and https://github.com/witnet/sheikah/issues/1476